tests/org.eclipse.passage.seal.demo.tests/OSGI-INF/anti-human-magic.product_0.2.1.keys_xmi

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="ASCII"?>
```

Comment:
This is a testing resource and must reside in _resource_ folder of curresponding testing bundle

Reply:
Done.